[BranchTime팀] django 초기세팅완료

nicholas019 opened the pull request:
## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [x] 기능 추가
- [ ] 리뷰 반영
- [ ] 리팩토링
- [ ] 버그 수정
- [ ] 컨벤션 수정

<br />

## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
1. django 초기세팅완료

<br />

## :: 구현 사항 설명 (작업한 내용을 상세하게 기록합니다.)
[세팅내용]
1. my_settings.py 생성, git에 업로드 되지 말아야할 SECRET_KEY, DATABASES, ALGORITHM 추가
2. core app생성 : timestamp 및 데코레이터를 위해 생성 
3. .gitignore 생성, git에 저장되지말아야될부분 정의 + my_settings.py추가
4. requirements.txt 생성 및 작성 : 설치된 패키지 정의
5. settings.py : 초기세팅 완료

<br />

## :: 성장 포인트 (해당 기능을 구현하며 고민했던 사항이나 새로 알게된 부분, 어려웠던 점 등을 작성합니다.)


<br />

## :: 기타 질문 및 특이 사항

nicholas019 committed:
[BranchTime팀] django 초기세팅완료

[세팅내용]
1. my_settings.py 생성, git에 업로드 되지 말아야할 SECRET_KEY, DATABASES, ALGORITHM 추가
2. core app생성
3. .gitignore 생성, git에 저장되지말아야될부분 정의 + my_settings.py추가
4. requirements.txt 생성 및 작성 : 설치된 패키지 정의
5. settings.py : 초기세팅 완료

lordmyshepherd commented on requirements.txt:
```diff
@@ -0,0 +1,8 @@
+mysqlclient==2.1.0
+PyMySQL==1.0.2
+PyJWT==2.4.0
+bcrypt==3.2.2
```
mysqlclient는 M1에서는 사용되지 않는 패키지입니다. 반대로 PyMySQL은 m1이 아닌 경우에 사용되지 않는 패키지입니다.

requirements.txt 파일을 배포용 패키지 파일, M1 개발환경용 패키지 파일 등으로 분리해 주세요.

```python
# requirements.txt 
Django==4.0.4
django-cors-headers==3.12.0
mysqlclient==2.1.0
```

```python
# requirements-dev.txt 
-r requirements.txt

ipython==7.25.0
django-extensions==3.1.1
```

```python
# requirements-m1-dev.txt 
Django==4.0.4
django-cors-headers==3.12.0
PyMySQL==1.0.2
django-extensions==3.1.1
```

lordmyshepherd commented:
추가로 file change에 보시면 `cpython-39.pyc`과 같은 .pyc 파일이 같이 올라와 있습니다.
이런 파일들이 올라오지 않도록 지우고 다시 커밋해서 
올려주세요!

nicholas019 committed:
[BranchTime팀] django 초기세팅수정

[세팅내용]
1. requirements.txt수정 : 배포용

2. requirements-dev.txt : 개발자용
3. requirements-m1-dev.txt : M1용 개발자용

nicholas019 commented:
수정완료했습니다

lordmyshepherd commented:
LGTM 👍🏻
